test/python/qobj/test_pulse_converter.py

```diff
@@ -354,6 +354,17 @@ def test_delay(self):
         self.assertEqual(converted_instruction.duration, instruction.duration)
         self.assertEqual(converted_instruction.instructions[0][-1], instruction)
 
+    def test_delay_memory_slot(self):
```

Comment:
Same. This test looks bit weird to me, but it should be okey if you have this use case in mind.

Reply:
No - this was just a way of fabricating directly what happens when you call pad() on an Acquire with a MemorySlot. So I will remove this as it isn't an appropriate use of the library.

Reply:
Now this test can be converted into `test_pad_no_delay_on_memory_slot`?